test/cluster-tests/kubernetes/test.sh

```diff
@@ -138,16 +138,20 @@ if (( ${running} != ${EXPECTED_DNS} )); then
 fi
 
 ###### Check for Kube-Dashboard
-log "Checking Kube-Dashboard"
-count=60
-while (( $count > 0 )); do
-  log "  ... counting down $count"
-  running=$(kubectl get pods --namespace=kube-system | grep kubernetes-dashboard | grep Running | wc | awk '{print $1}')
-  if (( ${running} == ${EXPECTED_DASHBOARD} )); then break; fi
-  sleep 5; count=$((count-1))
-done
-if (( ${running} != ${EXPECTED_DASHBOARD} )); then
-  log "K8S: gave up waiting for kubernetes-dashboard"; exit 1
+if (( ${EXPECTED_DASHBOARD} != 0 )); then
```

Comment:
LGTM. Does this test run with both dashboard enabled/disabled to verify the change?